cg/apps/demultiplex/sample_sheet/index.py

```diff
@@ -142,14 +146,22 @@ def pad_index_two(index_string: str, reverse_complement: bool) -> str:
     return index_string + INDEX_TWO_PAD_SEQUENCE
 
 
-def get_hamming_distance_for_indexes(sequence_1: str, sequence_2: str) -> int:
-    """Get the hamming distance between two index sequences.
+def get_hamming_distance_index_1(sequence_1: str, sequence_2: str) -> int:
+    """Get the hamming distance between two index 1 sequences.
     In the case that one sequence is longer than the other, the distance is calculated between
     the shortest sequence and the first segment of equal length of the longest sequence."""
     limit: int = min(len(sequence_1), len(sequence_2))
     return get_hamming_distance(str_1=sequence_1[:limit], str_2=sequence_2[:limit])
 
 
+def get_hamming_distance_index_2(sequence_1: str, sequence_2: str) -> int:
+    """Get the hamming distance between two index 2 sequences.
+    In the case that one sequence is longer than the other, the distance is calculated between
+    the shortest sequence and the last segment of equal length of the longest sequence."""
+    limit: int = min(len(sequence_1), len(sequence_2))
```

Comment:
```suggestion
    limit: int = [min(len(sequence_1), len(sequence_2))](limit: int = get_sequence_limit(sequence_1=sequence_1, sequence_2=sequence_2))
```

Reply:
the code: `limit: int = min(len(sequence_1), len(sequence_2))` is repeated below. It can be put in a function which can be called in both places.